src/Router/Controller/HomeController.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace App\Router\Controller;
+
+use App\Router\Request;
+
+class HomeController
+{
+    private string $titi;
+
+    public function __construct(string $titi)
+    {
+        $this->titi = $titi;
+    }
```

Comment:
Why this construct ? Where do you use $titi ?

Reply:
You're right, it was unused, just deleted it